Clinical-Genomics/cgweb
# Change RNA applications to wts

This PR changes all applications to be of prep-category wts

Install:
- [x] run dbcopy script for cg on clinical-db
- [x] install branch of [clinical-api](https://github.com/Clinical-Genomics/cg/pull/444) and cgweb on clinical-db
- [x] run the db-change script of the cg-branch

How to test RN applications of OF 1508:
1. login to OP-stage
1. create RNA order
1. add family
1. add sample

Expected outcome
- [x] only applications starting with RN show up in the application drop-down


How to test external RNA-applications of OF for external samples:
1. login to OP-stage
1. create external order
1. add family
1. add sample
1. open application option-list

Expected outcome
- [x] contains the RNA-application RNXCUSR000


This is a minor since we add functionality in a backwards compatible manner

- [x] code reviewed @ingkebil 
- [x] tested on stage @patrikgrenfeldt 
- [x] Deploy ready @ingkebil
